src/NavList/NavList.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import {ActionList} from '../ActionList'
+
+// ----------------------------------------------------------------------------
+// NavList
+
+export type NavListProps = {
+  children: React.ReactNode
+  'aria-label'?: string
+  'aria-labelledby'?: string
+  // sx
+}
+
+const Root = React.forwardRef<HTMLElement, NavListProps>(
+  ({'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, children}, ref) => {
+    return (
+      <nav ref={ref} aria-label={ariaLabel} aria-labelledby={ariaLabelledBy}>
```

Comment:
Should we pass down all `...props` for things like `id` / `data-test` etc?

Reply:
Sure, I'll pass down all props accepted by the `<nav>` element 👍

Reply:
Done ✅